### DISCUSSION TITLE
记录一下 WSL2 更换内核的过程 | ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#4] [Announcements]
# 记录一下 WSL2 更换内核的过程 | ネコのメモ帳

最近准备开始看 Kernel Pwn 了，但是 wsl2 的内核似乎并不是原版的，所以进行一个替换。

https://n.ova.moe/blog/2023/02/13/_wsl2%E7%9A%84%E5%86%85%E6%A0%B8%E5%AE%89%E8%A3%85

**Comment by MuelNova** [comment]
在这里，你需要注意一个问题。如果你更换了内核( 例如像文中的 6.19 )，那么在 WSL 内部安装 CUDA 后使用 nvidia-smi 将有可能报错（同时，PyTorch 等 使用 torch.cuda.is_available() 也将会返回 False）
